fission/src/systems/input/InputSystem.ts

```diff
@@ -18,8 +20,8 @@ abstract class Input {
         this.inputName = inputName
     }
 
-    /** @returns {number} a number between -1 and 1 for this input. */
-    abstract getValue(useGamepad: boolean): number
+    // Returns the current value of the input. Range depends on input type
+    abstract getValue(useGamepad: boolean, useTouchControls: boolean): number
```

Comment:
What if instead of `getValue()` taking in `useTouchControls`, the input scheme stored an enum InputType, replacing the boolean it currently has called `useGamepad`? That would avoid different input type conditions being passed in different ways and potentially overriding each other.

```ts
export enum InputType {
    KEYBOARD,
    GAMEPAD,
    TOUCH
 }

export type InputScheme = {
    schemeName: string
    descriptiveName: string
    customized: boolean
    inputType: InputType
    inputs: Input[]
}
```